tfm_ns_import.yaml

```diff
@@ -178,25 +190,25 @@
                 "dst": "tfm/platform/include/tfm_plat_ns.h"
             },
             {
-                "src": "../../tf-m-tests/test/test_services/tfm_secure_client_service/tfm_secure_client_service_api.h",
+                "src": "cmake_build/lib/ext/tfm_test_repo-src/test/test_services/tfm_secure_client_service/tfm_secure_client_service_api.h",
```

Comment:
Where does `cmake_build` come from? Would it be present if we use the `--cli 1` option?

Reply:
OK, this is from `build_tfm.py:585`:
```
585:    cmake_build_dir = join(TF_M_BUILD_DIR, "trusted-firmware-m", "cmake_build")
```